### DISCUSSION TITLE
how to get picture for capture recognition task

### DISCUSSION THREAD

**Discussion by jasperDD** [apache/incubator-mxnet#14110] [Q&A]
in this topic
https://github.com/apache/incubator-mxnet/tree/master/example/captcha
You can download the **images** and .rec files from https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/data/captcha_example.zip
there are no pictures in archive, how can i get this picture
please share it. if you have

**Comment by ChaiBapchya** [comment]
Its availalbe in the zip format. You can unzip it using the usual techniques

For Linux Ubuntu Terminal
```
apt-get install unzip
unzip /path/to/captcha-example.zip
```
For Mac, you could use the same `unzip` command
Hope this helps. 
In case it is resolved, you can go ahead and close it.

**Comment by ChaiBapchya** [comment]
@mxnet-label-bot add [Question]

**Comment by jasperDD** [comment]
I use windows) but   **it was typo , not now , and there are _no_ pictures in archive**
i.e in this link there are NO pictures, only rec and lst files

**Comment by Ishitori** [comment]
`Rec` file is a file that contains pictures. It is a special RecordIO format which allows to store many small binary files in one bigger file. It is done to increase speed of reading data from the disk. Use [ImageRecordDataset](https://mxnet.incubator.apache.org/versions/master/api/python/gluon/data.html#mxnet.gluon.data.vision.datasets.ImageRecordDataset) to open and use it.

If you want to know more about RecordIO format, take a look here:https://mxnet.incubator.apache.org/versions/master/architecture/note_data_loading.html
